jenkins/spark-premerge-build.sh

```diff
@@ -171,10 +171,10 @@ if [ -s "$M2_CACHE_TAR" ] ; then
 fi
 
 # Download a full version of spark
-$MVN_GET_CMD \
-    -DgroupId=org.apache -DartifactId=spark -Dversion=$SPARK_VER -Dclassifier=bin-hadoop3.2 -Dpackaging=tgz
+. jenkins/hadoop-def.sh $SPARK_VER
+wget -P $ARTF_ROOT https://archive.apache.org/dist/spark/spark-$SPARK_VER/spark-$SPARK_VER-$BIN_HADOOP_VER.tgz
```

Comment:
lets make it internal art-based as default, could simply use 

[SPARK_REPO](https://github.com/NVIDIA/spark-rapids/blob/branch-23.02/jenkins/version-def.sh#L46)
or $URM_URL